### ISSUE TITLE
[Feature] Not pull items when dropping them from your inventory.

### ISSUE THREAD

**Issue by Hazmatik** [SuperMartijn642/SimpleMagnets#19]
I see that this had been requested and implemented, sort of, the magnets don't pull items if you drop them from your hand, but if you open your inventory and drop them from it the items are still pulled towards the player. I tried both grabbing the items and tossing them out and hovering the mouse over the items and pressing the drop key and both ways the items are still pulled towards the player.

**Comment by SuperMartijn642** [comment]
I've looked into it a bit and it seems this is an issue in Forge. The item entity that spawns when an item is dropped from an inventory has no player id set.

I've created [a pull request](https://github.com/MinecraftForge/MinecraftForge/pull/7758) to fix the issue, so hopefully that gets merged.
If so, it would simply be fixed by updating to the newer Forge version without any changes on my end.
